src/views/Help/index.js

```diff
@@ -2,256 +2,279 @@ import React, { Component } from 'react';
-                You can come and go over the semester, and your Membership will move to your
-                official Co-Curricular Transcript only at the end of the semester.
-              </li>
-              <li>
-                <strong>What if I am in an Involvement I don&apos;t want to be part of? </strong>
-                No problem. Contact the Involvement leaders or advisor.
-              </li>
-              <li>
-                <strong>
-                  I&apos;m an alumna or alumnus. Why do I not see the join button anymore?{' '}
-                </strong>
-                Sorry, but alumni cannot join new Involvements. You could still subscribe to
-                Involvements to receive emails!
-              </li>
-            </Typography>
+          <Grid item xs={12} lg={8}>
+            <Card className="help">
+              <CardHeader
+                className="help-title"
+                title="Gordon 360 Help"
+                titleTypographyProps={{ variant: 'h4' }}
+              />
+              <CardContent>
+                <Card>
+                  <CardHeader
+                    className="help-header"
+                    title="LOGIN INSTRUCTIONS"
+                    titleTypographyProps={{ variant: 'body1' }}
+                  />
+                  <CardContent>
+                    <Typography variant="body1" component="ul">
+                      <li>Username: 'firstname.lastname' or your Gordon email address</li>
+                      <li>Password: Your Gordon College account password</li>
+                    </Typography>
+                  </CardContent>
+                </Card>
+
+                <Card>
+                  <CardHeader
+                    className="help-header"
+                    title="SITE NAVIGATION"
+                    titleTypographyProps={{ variant: 'body1' }}
+                  />
+                  <CardContent>
+                    <Typography variant="body1" component="ul">
+                      <li>
+                        <strong>Home</strong> &ndash; Contains various tiles with charts or information.
+                        <Typography variant="body1" component="ul">
+                          <li>
+                            <strong>Student:</strong> Includes your Dining “Meal Wheel’ tracker synced to days remaining in the
+                            semester and your Christian Life & Worship (CL&W) credits. You can click on the
+                            CL&W tracker to see which events are recorded for you.
+                          </li>
+                          <li>
+                            <strong>Faculty/Staff:</strong> Includes days remaining wheel and a separate Dining Dollars tracker.
+                            Also view your pending join requests for Involvements you’ve requested, as well
+                            as the requests that are pending for you approve if you are a Group Leader/Advisor.
```

Comment:
I wasn't sure about that. That was directly from CC's changes, but I have never looked at the homepage as a group advisor. Just remove this part you think?

Reply:
The code of the home page no longer includes that section, so it would never show up for anyone. Perhaps you can move the explanation to the Involvements page (since that is where the requests component now lives)?

Reply:
sounds good